project/plugins.sbt

```diff
@@ -1,4 +1,3 @@
-resolvers += Resolver.sonatypeRepo("releases")
```

Comment:
Removing this resolver will make that recent releases of _sbt-freestyle_ are not available until they are propagated to Maven Central (~2 hours sometimes). However, Sonatype ones are always available instantly.

Reply:
I removed it because it was provided by `sharedCommonSettings` in `sbt-org-policies`, like the bintray repo above. Though come to think of it, I'm not certain that the imported resolver setting will apply at the `project/` level; I tested locally but might have been using cache data.

Reply:
Discussed offline - yeah the resolver needs to stay. Will push a commit to fix this and also to update the `sbt-freestyle` dep once https://github.com/frees-io/sbt-freestyle/pull/59 is in.